src/XMonad/ManageHook.hs

```diff
@@ -65,7 +65,7 @@ infixr 3 <&&>, <||>
 
 -- | '||' lifted to a 'Monad'.
 (<||>) :: Monad m => m Bool -> m Bool -> m Bool
-(<||>) x y = ifM x (pure True) y
+(<||>) x = ifM x (pure True)
```

Comment:
This feels a bit weird

Reply:
should probably transform this to `x <||> y = ifM x (pure True) y` instead